apps/nouns-camp/src/hooks/dao-contract.js

```diff
@@ -280,9 +330,20 @@ export const useCreateProposal = () => {
 
     return writeContract({
       address: getContractAddress(chainId),
-      abi: parseAbi([
-        "function propose(address[] memory targets, uint256[] memory values, string[] memory signatures, bytes[] memory calldatas, string memory description) public returns (uint256)",
-      ]),
+      abi: [
+        {
+          inputs: [
```

Comment:
wonder if we care for the field names for some functions, or if we're ok with just the types everywhere, like:

```
inputs: [
   { type: "address[]" },
   { type: "uint256[]" },
   { type: "string[]" },
   { type: "bytes[]" },
   { type: "string" },
],
```